{{cookiecutter.project_shortname}}/.travis.yaml

```diff
@@ -27,9 +27,16 @@ services:
   - rabbitmq
 
 env:
-  - REQUIREMENTS=lowest E2E="no" ES_VERSION=6.0.0
-  - REQUIREMENTS=release DEPLOY=true E2E="yes" E2E_OUTPUT="base64" ES_VERSION=6.0.0
-  - REQUIREMENTS=devel E2E="yes" E2E_OUTPUT="base64" ES_VERSION=6.0.0
+  {%- if cookiecutter.elasticsearch == "5" %}
+  {%- set elasticsearch_test_version = "5.6" %}
+  {%- set elasticsearch_install_arg = "elasticsearch5" %}
+  {%- elif cookiecutter.elasticsearch == "6" %}
+  {%- set elasticsearch_test_version = "6.2" %}
```

Comment:
I think the download URL won't work with only the minor version specified. We can match the docker-services.yml version for ES and use `5.6.4` and `6.2.2`

Reply:
Indeed, I synced it with the docker-services.yml